Clinical-Genomics/scout
# Refactored and filtered general report data

Fix #2813.

- Filtered case key/values
- Filtered case individuals key/values
- Filtered institute key/values
- Filtered variants key/values (there is still a lot of information but it's required in the report)
- Modified variants list and arranged in this object:
variants = {
 "causatives_detailed" : [],
 "partial_causatives_detailed" : [],
 "suspects_detailed": [],
 "classified_detailed": [],
 "tagged_detailed": [],
 "tier_detailed": [],
 "dismissed_detailed": []
} 

**How to test**:
1. Install on stage and compare with branch master. General report produced with this branch should be exactly the same as the one produced by master branch.
2. Use the api to retrieve report data. It should contain all the required json info to produce the report

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
